src/ventures-for-shared-prosperity.njk

```diff
@@ -1,69 +0,0 @@
----
```

Comment:
@will0101 Even though we'll end up changing this content, let's keep the contents of this page so we have something for now. This file `ventures-for-shared-prosperity.njk` because the `ventures-for-shared-prosperity/index.njk` file. Same content, just different location and name.

Reply:
Understandable